src/pykeen/datasets/__init__.py

```diff
@@ -117,22 +117,26 @@ def get_dataset(
             logger.warning('dataset_kwargs not used since a pre-instantiated dataset was given')
         return dataset
 
-    if isinstance(dataset, str):
-        if has_dataset(dataset):
+    if isinstance(dataset, (str, pathlib.Path)):
+        if isinstance(dataset, str) and has_dataset(dataset):
```

Comment:
maybe better to start with an explicit check for` isinstance(pathlib.Path)`

Reply:
6061647